com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [12.2.0] - 2021-09-24
+## [12.3.0] - 2021-09-24
```

Comment:
you shouldn chant' the revision where the fix landed (even if I am agree we haven't promote this verison officially). Move ## [12.2.0] - 2021-09-24 up